backend/zeno_backend/processing/util.py

```diff
@@ -40,7 +45,10 @@ def generate_diff_cols(
 
     # various metadata type difference
     if diff_col_1.data_type == MetadataType.CONTINUOUS:
-        df.loc[:, "diff"] = df[col1_id] - df[col2_id]
+        if order_by == "descending":
+            df.loc[:, "diff"] = df[col1_id] - df[col2_id]
+        else:
+            df.loc[:, "diff"] = df[col2_id] - df[col1_id]
```

Comment:
Looks good but we should either do this, or add a check at the beginning of the function to make sure that the value of `order_by` is legal.
```suggestion
        elif order_by == "ascending":
            df.loc[:, "diff"] = df[col2_id] - df[col1_id]
        else:
            raise ValueError(f"Illegal value for {order_by=}")
```